### PR TITLE
Adds support for pixels in offset-anchor and offset-position.

### PR DESCRIPTION
**Pull request by courtneycb**

### CODE REVIEW

File: src/toTransform.js [hunk 1]
```diff
@@ -42,10 +42,10 @@
     if ('offset-position' in properties) {
       position = internalScope.offsetPositionAnchorParse(properties['offset-position']);
     }
-
     if (position === 'auto' || position === undefined || position === null) {
       return null;
     }
+    position = [position[0].value, position[1].value];
```
Comment (ewilligers) [inline src/toTransform.js]: When we have px, we need to skip the step of multiplying by offsetParent size / 100.

File: src/toTransform.js [hunk 2]
```diff
@@ -54,6 +54,8 @@
 
     if (anchor === 'auto' || anchor === undefined || anchor === null) {
       anchor = position;
+    } else {
+      anchor = [anchor[0].value, anchor[1].value];
```
Comment (ewilligers) [inline src/toTransform.js]: When we have px, we need to skip the step of multiplying by element size / 100.

File: src/toTransform.js [hunk 3]
```diff
@@ -74,11 +71,26 @@
...
-    var anchorPosY = (anchor[1] / 100) * elementProperties.height;
+    var anchorPosX = anchor[0].value;
+    var anchorPosY = anchor[1].value;
+    var offsetPosX = position[0].value;
+    var offsetPosY = position[1].value;
 
-    var offsetPosX = (position[0] / 100) * parentProperties.width;
-    var offsetPosY = (position[1] / 100) * parentProperties.height;
+    if (anchor[0].unit === '%') {
+      anchorPosX = (anchorPosX / 100) * elementProperties.width;
```
Comment (ewilligers) [inline src/toTransform.js]: For these 4 lines, can we divide as the last step.

File: src/toTransform.js (hunk 1 above)
Comment (courtneycb) [inline src/toTransform.js]: Done.

File: src/toTransform.js (hunk 2 above)
Comment (courtneycb) [inline src/toTransform.js]: Done.

File: src/toTransform.js (hunk 3 above)
Comment (courtneycb) [inline src/toTransform.js]: Done.

**Pull request merged**